### ISSUE TITLE
RStudio Desktop (Preview) do not open *.bib files

### ISSUE THREAD

**Issue by emilopezcano** [rstudio/rstudio#9797]
## Description of the problem

When clicking on a *.bib file (BibTeX database, plain text) on the files pane, the editor does not open it and an error message is shown ("File is binary rather than text...", see screenshot). I have not seen the problem reported in any other issue.

## Environment

I found the problem in the Preview version:

```
RStudio 2021.09.0-preview+338 "Ghost Orchid" Preview (802a0219d317c888a53f4fa6a527fc7bd1839446, 2021-08-23) for macOS
Mozilla/5.0 (Macintosh; Intel Mac OS X 11_5_2) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.10 Chrome/69.0.3497.128 Safari/537.36
```

And it works fine in the latest release (Version 1.4.1717)

Thanks for your work!

<img width="667" alt="Captura de pantalla 2021-09-01 a las 8 15 49" src="https://user-images.githubusercontent.com/666708/131621411-a7417968-a0e9-4b50-9dbe-87b2f911d6af.png">

**Comment by ronblum** [comment]
@emilopezcano Thank you for raising the issue!

@astayleraz I can't reproduce it on MacOS 12 beta. Can you check whether this is an issue in MacOS 11.5.x?

**Comment by astayleraz** [comment]
@ronblum I was not able to reproduce this with RStudio 2021.09.0-preview+340 "Ghost Orchid" Preview on macOS 11.5.2. Im curious @emilopezcano if you see this with other .bib files or it is something specific to the one you are using / have on your machine?

**Comment by emilopezcano** [comment]
Yes, I saw that with two different .bib files in two different RStudio projects. It was with build 338 of 2021.09.0

I have tried the following things now:

1. Create a new blank project, create a blank text file, paste a bibtex entry and save as .bib. Close the file, when clicking on the files pane, the same error.
2. The same, after installing build 340 -> the same error
3. Try again in stable version 1.4.1717 -> the same error, but it worked yesterday

My guess now is that it could be something related with having several versions of RStudio installed on MacOS (see screen capture)

PS: I also tried fresh install of R and RStudio in Windows -> Works

<img width="229" alt="image" src="https://user-images.githubusercontent.com/666708/131798289-042f5633-626f-4536-8f73-5490ce7d5d20.png">

**Comment by emilopezcano** [comment]
No longer happened

**Comment by ronblum** [comment]
@emilopezcano Thank you for the update!
